src/aerovaldb/utils/uri.py

```diff
@@ -85,6 +102,11 @@ def parse_uri(uri: str) -> tuple[str, dict[str, str], dict[str, str]]:
 
 
 def build_uri(route: str, route_args: dict, kwargs: dict = {}) -> str:
+    for k, v in route_args.items():
+        encoded = v.replace('"', '\\"')
```

Comment:
could you use a raw-string here instead of escaping `\`?

Reply:
And maybe an explanation what you want to escape and why?

Reply:
> could you use a raw-string here instead of escaping `\`?

Tried it but it seems that backslash as the final character in a raw string literal still escapes the quote. `r"\"` doesn't work.